pallets/aleph/src/migrations/v1_to_v2.rs

```diff
@@ -0,0 +1,49 @@
+use crate::Config;
+use frame_support::log;
+use frame_support::{
+    generate_storage_alias,
+    traits::{Get, GetStorageVersion, PalletInfoAccess, StorageVersion},
+    weights::Weight,
+};
+
+generate_storage_alias!(Aleph, SessionForValidatorsChange => Value<()>);
+generate_storage_alias!(Aleph, MillisecsPerBlock => Value<()>);
+generate_storage_alias!(Aleph, SessionPeriod => Value<()>);
+
+pub fn migrate<T: Config, P: GetStorageVersion + PalletInfoAccess>() -> Weight {
+    let on_chain_storage_version = <P as GetStorageVersion>::on_chain_storage_version();
+    let current_storage_version = <P as GetStorageVersion>::current_storage_version();
+
+    if on_chain_storage_version == 1 && current_storage_version == 2 {
+        log::info!(target: "pallet_aleph", "Running migration from STORAGE_VERSION 1 to 2");
+
+        if !SessionForValidatorsChange::exists() {
+            log::info!(target: "pallet_aleph", "Storage item SessionForValidatorsChange does not exist!");
+        }
+        SessionForValidatorsChange::kill();
+
+        if !MillisecsPerBlock::exists() {
+            log::info!(target: "pallet_aleph", "Storage item MillisecsPerBlock does not exist!");
+        }
+        MillisecsPerBlock::kill();
+
+        if !SessionPeriod::exists() {
+            log::info!(target: "pallet_aleph", "Storage item SessionPeriod does not exist!");
+        }
+        SessionPeriod::kill();
+
+        // store new version
+        StorageVersion::new(2).put::<P>();
+
+        T::DbWeight::get().reads(1) + T::DbWeight::get().writes(1)
+    } else {
+        log::warn!(
```

Comment:
why `warn` here?

Reply:
We don't want to see it by default on info level.